README.md

```diff
@@ -1,2 +1,18 @@
-# daccs-project-website
-Information site about the DACCS project
+# The DACCS Project Website
+This site is to give a summary of what the DACCS project is about.  It includes a mission statement, vision statement, a list of the Principle Investigators and a list of the Funders.
+
+# Building the site
+
+Clone the repository at [https://github.com/DACCS-Climate/daccs-project-website](https://github.com/DACCS-Climate/daccs-project-website)
```

Comment:
Not needed, they're already on that page

Reply:
Removed URL